pkg/gvk/gvk.go

```diff
@@ -113,28 +112,26 @@ var order = []string{
 	"CronJob",
 	"PodDisruptionBudget",
 }
+var orderLast = []string{
+	"ValidatingWebhookConfiguration",
+}
 var typeOrders = func() map[string]int {
 	m := map[string]int{}
-	for i, n := range order {
-		m[n] = i
+	for i, n := range orderFirst {
+		m[n] = -len(orderFirst) + i
+	}
```

Comment:
so all the orderFirst are negatives, and orderlasts positive and unrecognized get zero so sit in the middle?
nice

Reply:
yes